mandatory/3-playing-computer.js

```diff
@@ -40,3 +40,18 @@ for (let i = 0; i < 5; ++i) {
     console.log(e);
   }
 }
+// 1. This program throws an error. Why? (If you can't find it, try executing it).
+// x is not defined
+// 2. Remove the line that throws the error.
+// line 18
+// 3. What is printed to the console?
+//
+// 4. How many times is "f1" called?
+// 2
+// 5. How many times is "f2" called? 
+// 2
+// 6. What value does the "a" parameter take in the first "f1" call?
+// 7
```

Comment:
This is a really bad question in my opinion that's main purpose seems to be to be confusing. Basically what it's asking is, if we look at f1:
const f1 = function (a, b) {
  return a + b;
};

What value is being put in as the first parameter of this function when it reaches it the first time. So it wants to know what the a value in this function is rather than the "global" a value that you suggested.

Like I said, it's a very annoying question, I don't really like it that much so don't worry about this one too much